blueprint.md

```diff
@@ -1,57 +1,37 @@
-# How To Use This Template
-This document describes how to use this transformer template for your custom algorithm processing plot-level RGB data.
+# [{{pkg.name}}]({{ pkg.url }})
 
-## Assumptions
-It is assumed that:
-- you are generating a Docker image containing your algorithm and that you have Docker installed on your computer
-- are familiar with GitHub template repositories, or know how to use `git`
-
-## Steps to take
-The following steps can be taken to develop your algorithm for inclusion into a processing pipeline.
-
-1. [Setup](#setup): Click the `Use this template` button in GitHub to make a copy of this repository (or run `git clone`)
-2. [Definitions](#definitions): Fill in and modify the definitions in the algorithm_rgb.py file
-3. [Algorithm](#algorithm): Replace the code in the `calculate` function with your algorithm
-4. [Generate](#generate): Run `generate.py` to create a Dockerfile
-5. [Test](#test): Run the `testing.py` script to run your algorithm and validate the results
-6. [Docker](#build_docker): Create a Docker image for your algorithm and publish it
-7. [Testing Your docker image](#test_docker): OPTIONAL
-8. [Testing Image Production](#production): OPTIONAL
-9. [Finishing](#finishing): Finish up your development efforts
+## Description
+{{ pkg.algorithm_description }}
 
-### Setup your repo <a name="setup"/>
-The first thing to do is to create a copy of this repository has a meaningful name and that you are able to modify.
-In GitHub this is easy, browse to this [repository](https://github.com/AgPipeline/template-rgb-plot) and click the `Use this template` button.
-You will be led through the steps necessary to create a clone in a location of your choosing.
+### Methods
 
-If you are not on GitHub, you will need to setup your `git` environment and clone the repository.
+{{ pkg.methods }}
 
-### Fill in your definitions <a name="definitions" />
-To fill in the needed definitions, first open the `algorithm_rgb.py` file in your favorite editor.
+## Assumptions
+It is assumed that:
 
-If you are modifying your existing code, you should consider updating the version number definition: `VERSION`.
-It's assumed that [Semantic Version numbers](https://semver.org/) will be used, but any methodology can be used.
+* An image folder is located in the root of this directory that will provide images for the calculate()
+function to process. Sample plot images can be found [here](https://drive.google.com/file/d/1xWRU0YgK3Y9aUy5TdRxj14gmjLlozGxo/view)
```

Comment:
Same as in README.md, replace Google Drive with CyVerse URL: https://de.cyverse.org/dl/d/4108BB75-AAA3-48E1-BBD4-E10B06CADF54/sample_plot_images.zip and include instructions on how to download and extract using shell command (use wget and unzip for example)